### ISSUE TITLE
Deprecated %kernel.root_dir%

### ISSUE THREAD

**Issue by Vincz** [overblog/GraphQLBundle#623]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | no
| RFC?             | no
| Version/Branch   | latests

With symfony 5, the container parameter `kernel.root_dir` is now deprecated. 
It is used in the `commands.yaml` and also [here](https://github.com/overblog/GraphQLBundle/blob/38db44132cdd491efe6b36723fed9aaff670353a/src/DependencyInjection/Compiler/ConfigParserPass.php#L165).

**Comment by mcg-web** [comment]
Oops I missed this one in 0.11, I'm on it!

**Comment by mcg-web** [comment]
[This one is normal] (https://github.com/overblog/GraphQLBundle/blob/38db44132cdd491efe6b36723fed9aaff670353a/src/DependencyInjection/Compiler/ConfigParserPass.php#L165) we don't need this with flex. in commands.yaml we replace it with `%kernel.cache_dir%/../..` since this is just a default option so it can be override.

**Comment by mcg-web** [comment]
@Vincz can you give it a try with `0.12@dev` please?

**Comment by Vincz** [comment]
Stupid question, but in composer I'm in minimum stability dev, and try to install the 0.12 with 
`"overblog/graphql-bundle": "dev-0.12"`
But it fails. How can I update to use the `0.12@dev` as you suggest ?

**Comment by mcg-web** [comment]
you must add `@dev` behind your requirement like that `"overblog/graphql-bundle": "0.12.*@dev"`

**Comment by Vincz** [comment]
Event with `"overblog/graphql-bundle": "0.12.*@dev"`,  It keeps installing me the 0.12.4. I cleared the cache also. 

```
Package operations: 1 install, 0 updates, 0 removals
  - Installing overblog/graphql-bundle (v0.12.4): Downloading (100%)
```

**Comment by mcg-web** [comment]
Try with `composer req 'overblog/graphql-bundle:0.12.*@dev'` ?

**Comment by Vincz** [comment]
So, I finally managed to install the specified version with :
`composer req 'overblog/graphql-bundle:0.12.x-dev'`

It seems to work with your last commit ! 😺

**Comment by mcg-web** [comment]
Nice thank you, I'll release a new version for 0.12 later .

**Comment by mcg-web** [comment]
Released in v0.12.5, thank you for report :+1:
